.github/workflows/ci.yml

```diff
@@ -70,3 +70,15 @@ jobs:
       run: cargo test cli_tests
     - name: Build with release flag
       run: cargo build --release --all-features
+  audit-latest:
+    name: Audit latest dependencies
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+    - name: Install rust toolchain
+      run: rustup set profile minimal
+    - name: Install cargo audit
+      run: cargo install cargo-audit
+    - name: Run audit command
+      run: cargo audit
```

Comment:
Could you add the `-D` options and `--quiet` option as well as #211? Based on my testing in https://github.com/StoneDot/dynein/pull/22, I found that we can configure the repository to allow merge a pull request with partial test failures.

I think display as a failure in case of warnings, unmaintained, unsound or yanked is more appropriate because we can notice them before merging it.